lua/nvim-tree.lua

```diff
@@ -11,6 +11,16 @@ local M = {
   init_root = "",
 }
 
+--- Helper function to execute some explorer method safely
+---@param fn string # key of explorer
+---@return function|nil
+local function explorer_fn(fn)
+  local explorer = core.get_explorer()
+  if explorer then
+    return explorer[fn]
+  end
+end
```

Comment:
Love your work. This isolates these cases so that they may be more easily identified and resolved as the multi instance project progresses.